iam.tf

```diff
@@ -0,0 +1,86 @@
+module "iam_label" {
+  source    = "github.com/cloudposse/tf_label"
+  namespace = "${var.namespace}"
+  stage     = "${var.stage}"
+  name      = "${var.name}"
+}
+
+data "aws_iam_policy_document" "ec2" {
+  statement {
+    sid     = "EC2AssumeRole"
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals = {
+      type        = "Service"
+      identifiers = ["ec2.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "ec2_role" {
+  name               = "efs-backup-ec2-role"
+  assume_role_policy = "${data.aws_iam_policy_document.ec2.json}"
+}
+
+resource "aws_iam_instance_profile" "instance_profile" {
+  name = "efs-backup-instance-profile"
+  role = "${aws_iam_role.ec2_role.name}"
+}
+
+data "aws_iam_policy_document" "datapipeline_resource" {
+  statement {
+    sid     = "EC2AssumeRole"
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals = {
+      type        = "Service"
+      identifiers = ["ec2.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "datapipeline_resource" {
+  name               = "efs-backup-datapipeline-resource-role"
```

Comment:
this needs to use a label. drop `-resource-role` suffix as it does not add information.